yarn.lock

```diff
@@ -1008,6 +1008,10 @@ find-index@^0.1.1:
   version "0.1.1"
   resolved "https://registry.yarnpkg.com/find-index/-/find-index-0.1.1.tgz#675d358b2ca3892d795a1ab47232f8b6e2e0dde4"
 
+find-root@^0.1.1:
```

Comment:
is this still necessary? Seems like changes to this file should have been dropped since no changes to root package.json